### ISSUE TITLE
Setting cookie name example is wrong

### ISSUE THREAD

**Issue by ghyde** [openshift/openshift-docs#31529]
### Which section(s) is the issue in?

_Networking > Routes > Route configuration > Annotating a route with a cookie_

This issue applies to all 4.x documentation.

[Source][src]

### What needs fixing?

The route annotation example is not valid. I tested the given example, and it does not overwrite the cookie name. The annotation must always use the key `router.openshift.io/cookie_name` ([Reference][ref]). For example, the documentation should use:

```
oc annotate route my_route router.openshift.io/cookie_name="my_cookie"
```

[src]: https://github.com/openshift/openshift-docs/blob/f004e3037811168c4d9404311b55e5f29ad4dca3/modules/nw-annotating-a-route-with-a-cookie-name.adoc
[ref]: https://docs.openshift.com/container-platform/4.7/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration

**Comment by jboxman** [comment]
cc @bmcelvee

**Comment by ghyde** [comment]
@jboxman @bmcelvee Any update on this issue? The incorrect documentation is negatively affecting OCP 4 users.

I already submitted a PR fixing this issue (#31530).

**Comment by jboxman** [comment]
@ghyde, I asked QE to confirm and then we'll get this merged, thanks!

**Comment by ghyde** [comment]
This issue was resolved with #31530.
